crates/ruma-events/src/matrix_rtc/focus.rs

```diff
@@ -0,0 +1,85 @@
+//! Types for matrixRTC Focus/SFU configurations.
+
+use serde::{Deserialize, Serialize};
+
+/// Description of the SFU/Focus a membership can be connected to.
+///
+/// A focus can be any server powering the matrixRTC session (SFU,
+/// MCU). It serves as a node to redistribute RTC streams.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[cfg_attr(test, derive(PartialEq))]
```

Comment:
Why only for tests? It seems okay to derive that all the time (with `Eq`, of course).